content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -1177,6 +1183,33 @@ The `fieldPath` field does not support indexing arrays numerically.
 
 Setting `fieldPath` is optional.
 
+#### The `optionalOldSelf` field {#field-optional-oldself}
+
+The `optionalOldSelf` field is a boolean field added in Kubernetes 1.29. The feature
```

Comment:
Use a `{{< feature-state state="alpha" for_k8s_version="v1.29" >}}` tag here to describe stability instead of threading it into the text description? (It's easier to remove later..

Reply:
added